pir/proto/payload.proto

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2020 the authors listed in CONTRIBUTORS.md
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+syntax = "proto2";
+
+package pir;
+/*
+#include "pir/proto/pryload.pb.h"
+*/
+
+/*PayloadData holds the request/response data buffers, usually ciphertexts */
+message PayloadData {
+  repeated bytes data = 1; //query or response data
+}
+
+/*Payload contains the entire request or response message */
+message Payload {
```

Comment:
As above, let's name this more specifically. What is the payload for? So far as I can tell, this is only ever used for a Request. So why not name it as such? The more direct the name, the clearer your intentions are.